gordo/machine/model/anomaly/diff.py

```diff
@@ -72,7 +72,10 @@ def get_metadata(self):
             metadata[
                 "feature-thresholds-per-fold"
             ] = self.feature_thresholds_per_fold_.to_dict()
-
+        if hasattr(self, "aggregate_threshold_per_fold_"):
+            metadata[
+                "aggregate-thresholds-per-fold"
+            ] = self.aggregate_threshold_per_fold_
```

Comment:
small point here, but to remain consistent, maybe rename this to also be in plural form. `aggregate_thresholds_per_fold_`

Reply:
Yeah, correctly spotted.